### PR TITLE
Need to check the mF2C Docker-compose file

### PR DESCRIPTION
**Pull request by ssgUPC**
@cjdcordeiro and @ALEJANDROJ19 please check the modified docker-compose file in my branch and if you think it is correct then, please accept the merge request

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -252,48 +148,49 @@ services:
...
       - HOST_IP=
+      - DATACLAY_EP=
       - WORKING_DIR_VOLUME=/tmp/compose_files
     volumes:
       - /tmp/compose_files:/tmp/compose_files
-      - /var/run/docker.sock:/var/run/docker.sock  
-  resouce-categorization:
+      - /var/run/docker.sock:/var/run/docker.sock
+  resource-categorization:
     image: mf2c/resource-categorization:latest
```
Comment (cjdcordeiro) [inline docker-compose/docker-compose.yml]: pls do not use `latest`

**Comment by ALEJANDROJ19** [comment]
Solved a conflict in the file. Now it's correct.

**Comment by cjdcordeiro** [comment]
@ssgUPC the `device` now seems to be pushed into CIMI, but it fails:

```resource-categorization_1  | Dynamic: {"device": "None", "ramFree": 90.2578125, "ramFreePercent": 2.299999999999997, "storageFree": 23691.5859375, "storageFreePercent": 41.7, "cpuFreePercent": 45.3, "powerRemainingStatus": "The device has connected with Electricity", "powerRemainingStatusSeconds": "The device has connected with Electricity", "ethernetAddress": "[snicaddr(family=<AddressFamily.AF_INET: 2>, address='172.25.0.21', netmask='255.255.0.0', broadcast='172.25.255.255', ptp=None), snicaddr(family=<AddressFamily.AF_PACKET: 17>, address='02:42:ac:19:00:15', netmask=None, broadcast='ff:ff:ff:ff:ff:ff', ptp=None)]", "wifiAddress": "Empty", "ethernetThroughputInfo": [8187, 6402, 59, 83, 0, 0, 0, 0], "wifiThroughputInfo": ["E", "m", "p", "t", "y"], "myLeaderID": "None"}
cimi_1                     | java.lang.ClassCastException: java.lang.String cannot be cast to clojure.lang.IPersistentMap
cimi_1                     | 	at clojure.lang.RT.dissoc(RT.java:884)
cimi_1                     | 	at clojure.core$dissoc.invokeStatic(core.clj:1503)
cimi_1                     | 	at clojure.core$dissoc.invokeStatic(core.clj:1496)
cimi_1                     | 	at clojure.core$dissoc.doInvoke(core.clj:1496)
cimi_1                     | 	at clojure.lang.RestFn.invoke(RestFn.java:521)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.resources.common.utils$strip_service_attrs.invokeStatic(utils.clj:74)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.resources.common.utils$strip_service_attrs.invoke(utils.clj:69)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.resources.common.std_crud$add_fn$fn__3342.invoke(std_crud.clj:30)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.resources.device_dynamic$eval22103$fn__22104.invoke(device_dynamic.clj:43)
cimi_1                     | 	at clojure.lang.MultiFn.invoke(MultiFn.java:229)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.app.routes$fn__2638$fn__2639.invoke(routes.clj:17)
cimi_1                     | 	at compojure.core$wrap_response$fn__2488.invoke(core.clj:158)
cimi_1                     | 	at compojure.core$wrap_route_middleware$fn__2472.invoke(core.clj:128)
cimi_1                     | 	at compojure.core$wrap_route_info$fn__2477.invoke(core.clj:137)
cimi_1                     | 	at compojure.core$wrap_route_matches$fn__2481.invoke(core.clj:146)
cimi_1                     | 	at compojure.core$routing$fn__2496.invoke(core.clj:185)
cimi_1                     | 	at clojure.core$some.invokeStatic(core.clj:2693)
cimi_1                     | 	at clojure.core$some.invoke(core.clj:2684)
cimi_1                     | 	at compojure.core$routing.invokeStatic(core.clj:185)
cimi_1                     | 	at compojure.core$routing.doInvoke(core.clj:182)
cimi_1                     | 	at clojure.lang.RestFn.applyTo(RestFn.java:139)
cimi_1                     | 	at clojure.core$apply.invokeStatic(core.clj:659)
cimi_1                     | 	at clojure.core$apply.invoke(core.clj:652)
cimi_1                     | 	at compojure.core$routes$fn__2500.invoke(core.clj:192)
cimi_1                     | 	at compojure.core$routing$fn__2496.invoke(core.clj:185)
cimi_1                     | 	at clojure.core$some.invokeStatic(core.clj:2693)
cimi_1                     | 	at clojure.core$some.invoke(core.clj:2684)
cimi_1                     | 	at compojure.core$routing.invokeStatic(core.clj:185)
cimi_1                     | 	at compojure.core$routing.doInvoke(core.clj:182)
cimi_1                     | 	at clojure.lang.RestFn.applyTo(RestFn.java:139)
cimi_1                     | 	at clojure.core$apply.invokeStatic(core.clj:659)
cimi_1                     | 	at clojure.core$apply.invoke(core.clj:652)
cimi_1                     | 	at compojure.core$routes$fn__2500.invoke(core.clj:192)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.middleware.cimi_params$wrap_cimi_params$fn__2813.invoke(cimi_params.clj:59)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.middleware.base_uri$wrap_base_uri$fn__2707.invoke(base_uri.clj:40)
cimi_1                     | 	at ring.middleware.keyword_params$wrap_keyword_params$fn__3812.invoke(keyword_params.clj:36)
cimi_1                     | 	at ring.middleware.nested_params$wrap_nested_params$fn__3870.invoke(nested_params.clj:89)
cimi_1                     | 	at ring.middleware.params$wrap_params$fn__3944.invoke(params.clj:67)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.middleware.exception_handler$wrap_exceptions$fn__2937.invoke(exception_handler.clj:18)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.middleware.authn_info_header$wrap_authn_info_header$fn__2025.invoke(authn_info_header.clj:89)
cimi_1                     | 	at metrics.ring.expose$expose_metrics_as_json$fn__3562.invoke(expose.clj:138)
cimi_1                     | 	at ring.middleware.json$wrap_json_body$fn__3747.invoke(json.clj:44)
cimi_1                     | 	at ring.middleware.json$wrap_json_response$fn__3772.invoke(json.clj:87)
cimi_1                     | 	at metrics.ring.instrument$handle_request$fn__3582.invoke(instrument.clj:65)
cimi_1                     | 	at metrics.ring.instrument.proxy$java.lang.Object$Callable$7da976d4.call(Unknown Source)
cimi_1                     | 	at com.codahale.metrics.Timer.time(Timer.java:101)
cimi_1                     | 	at metrics.ring.instrument$handle_request.invokeStatic(instrument.clj:64)
cimi_1                     | 	at metrics.ring.instrument$handle_request.invoke(instrument.clj:54)
cimi_1                     | 	at metrics.ring.instrument$instrument$fn__3590.invoke(instrument.clj:91)
cimi_1                     | 	at com.sixsq.slipstream.ssclj.middleware.logger$wrap_logger$fn__2965.invoke(logger.clj:60)
cimi_1                     | 	at ring.middleware.cookies$wrap_cookies$fn__3716.invoke(cookies.clj:175)
cimi_1                     | 	at aleph.http.server$handle_request$fn__14348$f__4381__auto____14349.invoke(server.clj:157)
cimi_1                     | 	at clojure.lang.AFn.run(AFn.java:22)
cimi_1                     | 	at io.aleph.dirigiste.Executor$Worker$1.run(Executor.java:62)
cimi_1                     | 	at manifold.executor$thread_factory$reify__4263$f__4264.invoke(executor.clj:44)
cimi_1                     | 	at clojure.lang.AFn.run(AFn.java:22)
cimi_1                     | 	at java.lang.Thread.run(Thread.java:748)
resource-categorization_1  | r <Response [500]> <PreparedRequest [POST]> Internal Server Error```

there's some data type issue in there. Please make sure your schemas are consistent

**Comment by ssgUPC** [comment]
@cjdcordeiro let me check

**Pull request closed without merging**